lib/osf-components/addon/components/node-picker/template.hbs

```diff
@@ -0,0 +1,27 @@
+{{#if this.findNodes.last}}
+    <div data-test-collections-item-picker local-class='form-group'>
+        <PowerSelect
+            @renderInPlace={{false}}
+            @searchEnabled={{true}}
+            @placeholder={{t 'move_to_project.select_placeholder'}}
```

Comment:
Depending on what we what we want to show, you may want to update the translation used here